Basic/Basic/OrganizationInfoViewController.swift

```diff
@@ -33,4 +37,47 @@ class OrganizationInfoViewController: UIViewController
         alert.addAction(okAction);
         self.present(alert, animated: true, completion: nil);
     }
+    
+    @IBAction func clickFetchOrganizations(_ sender: UIButton)
+    {
+        let appDel = AppDelegate.instance()
+        let context = appDel!.persistentContainer.viewContext
+        let coord = appDel?.persistentContainer.persistentStoreCoordinator
+        
+        let fetchRequest = NSFetchRequest<NSFetchRequestResult>(entityName: "OrganizationModel")
+        let deleteRequest = NSBatchDeleteRequest(fetchRequest: fetchRequest)
```

Comment:
You shouldn't delete from the DB until the request completes. What if the request fails?